### PR TITLE
Add documentacion regarding Group management

### PR DESCRIPTION
**Pull request by saraycp**

### CODE REVIEW

**Comment by saraycp** [comment]
[Documentation](https://trello.com/c/KZwAiKTy/1069-documentation)

File: xml/obs_ag_user_management.xml [hunk 1]
```diff
@@ -107,6 +107,14 @@
    confirmation is needed after registration before the user may
    login.</para>
  </sect2>
+ <sect2 xml:id="_users_and_group_maintainers">
+  <title>Users and Group Maintainers</title>
+  <para>Administrators can create new groups and add and remove users on it, as
+    well as give the users Maintainer rights. This way, a mantainer user will
```
Comment (dmarcoux) [inline xml/obs_ag_user_management.xml]: Administrators can create <strike>new</strike> groups, add users to them , remove users from them and give Maintainer rights to users. This way, a ma**i**ntainer will (...)

File: xml/obs_basic_workflow.xml [hunk 2]
```diff
@@ -735,4 +735,12 @@ At revision 1.</screen>
   </sect2>
  </sect1>
 
+ <sect1 xml:id="sec.obs.basicworkflow.manage-group">
+  <title>Manage Group</title>
+  <para>
+    Users with the role Maintainer can add and remove users from its group. They
```
Comment (dmarcoux) [inline xml/obs_basic_workflow.xml]: Users with Maintainer rights  can add users to their group and remove users from it.

File: xml/obs_best_practice_manage_group.xml [hunk 3]
```diff
@@ -0,0 +1,29 @@
...
+  <!ENTITY % entities SYSTEM "entity-decl.ent">
+    %entities;
+]>
+<sect1 version="5.1" xml:id="manage_group"
+ xmlns="http://docbook.org/ns/docbook"
+ xmlns:xi="http://www.w3.org/2001/XInclude"
+ xmlns:xlink="http://www.w3.org/1999/xlink" >
+  <title>Manage Group</title>
+  <para>
+    Only administrators and users with the role Maintainer can manage groups.
```
Comment (dmarcoux) [inline xml/obs_best_practice_manage_group.xml]: with Maintainer rights

File: xml/obs_best_practice_manage_group.xml [hunk 4]
```diff
@@ -0,0 +1,29 @@
...
+    They can add and remove other users from the group, as well as give them
+    Maintainer's rights.
```
Comment (dmarcoux) [inline xml/obs_best_practice_manage_group.xml]: Maintainer<strike>'s</strike>

File: xml/obs_best_practice_manage_group.xml [hunk 5]
```diff
@@ -0,0 +1,29 @@
...
+  </para>
+  <para>
+    On the Group Members tab there is a link to Add User, then enter the name
```
Comment (dmarcoux) [inline xml/obs_best_practice_manage_group.xml]: On the group Members tab<b>, (missing comma)</b>, there is a link to Add **Member**

File: xml/obs_best_practice_manage_group.xml [hunk 6]
```diff
@@ -0,0 +1,29 @@
...
+    of an existing user. You can click on the Maintainer checkbox to add that
```
Comment (dmarcoux) [inline xml/obs_best_practice_manage_group.xml]: You can click on the Maintainer checkbox to give Maintainer rights to a user.

**Comment by saraycp** [comment]
@dmarcoux  all the feedback addressed, thanks!

**Pull request merged**